iceoryx2-bb/lock-free/src/spmc/unrestricted_atomic.rs

```diff
@@ -38,6 +38,8 @@ use std::{
     sync::atomic::{AtomicBool, AtomicU64, Ordering},
 };
 
+// ATTENTION: To ensure the functionality also in the case of an overflow wot the 'write_cell'
```

Comment:
What means `wot`?

Reply:
That's scotch for "what". But you are right, it doesn't make sense in this context and should be "wi"